test_suites/cvm/cvm_test.go

```diff
@@ -218,7 +218,17 @@ func TestTDXAttestation(t *testing.T) {
 			if err != nil {
 				t.Fatalf("kernelRev, err := strconv.Atoi(kernelRevStr): %v, want nil", err)
 			}
-			if int(kernelRev) >= 1016 {
+
+			// Kernel revisions assuming jammy image
+			lowerKernelRev := 1016
+			upperKernelRev := 1021
+
+			if strings.Contains(image, "noble") {
+			    lowerKernelRev = 1006
+			    upperKernelRev = 1008
+			}
+
+			if !strings.Contains(image, "oracular") && int(kernelRev) >= lowerKernelRev && int(kernelRev) < upperKernelRev {
```

Comment:
@a-crate I have added kernel specific checks. However, the downside is that this will have to be maintained as new Ubuntu versions are released and older ones EOL

Reply:
Can we assume that anything newer than noble already has the tdx_guest module available and skip the package installation? In that case you can just check if it's jammy/oracular/noble, if so then check kernel and perform package installation.

Also it would be preferable from our side to check for numeric version rather than codename, as the codename doesn't always get preserved in derivative images for test projects.

Reply:
Addressed this feedback in the latest push. Could you please take another look?